pydm/tests/widgets/test_enum_button.py

```diff
@@ -174,6 +174,58 @@ def test_check_enable_state(qtbot, connected, write_access, has_enum,
         assert "Enums not available" in actual_tooltip
 
 
+def test_items_is_not_none():
+    """
+    Test that items property is not None, as QStringList definition meant for Qt Designer cannot handle that.
+    """
+    widget = PyDMEnumButton()
```

Comment:
Missing here the `qtbot` and `qtbot.addWidget(widget)`.

Reply:
It shouldn't matter here, but I can add it for the sake of consistency.